src/lib/cert/cvc/ecdsa_sig.cpp

```diff
@@ -50,7 +50,7 @@ ECDSA_Signature decode_concatenation(const std::vector<byte>& concat)
 
    const size_t rs_len = concat.size() / 2;
 
-   BigInt r = BigInt::decode(&concat[0], rs_len);
+   BigInt r = BigInt::decode(concat.data(), rs_len);
```

Comment:
Why change this? concat is of a known size, and we follow up &x[0] with &x[n] - if &x[0] was a problem the latter index certainly will be as well. And it makes it harder to understand where the data is being placed, as two distinct syntaxes are used.